internal/controller/controllers/agent_controller.go

```diff
@@ -1846,16 +1849,17 @@ func createNewHost(agent *v1beta1.Agent, clusterID *strfmt.UUID, infraEnvID strf
 	// Create Host model
 	hostID := strfmt.UUID(agent.Name)
 	host := &models.Host{
-		ID:            &hostID,
-		Kind:          swag.String(models.HostKindHost),
-		RegisteredAt:  strfmt.DateTime(agent.CreationTimestamp.Time),
-		CheckedInAt:   strfmt.DateTime(agent.CreationTimestamp.Time),
-		Role:          role,
-		SuggestedRole: role,
-		ClusterID:     clusterID,
-		InfraEnvID:    infraEnvID,
-		Status:        &hostStatus,
-		Inventory:     inventory,
+		ID:              &hostID,
+		Kind:            swag.String(models.HostKindHost),
+		RegisteredAt:    strfmt.DateTime(agent.CreationTimestamp.Time),
+		CheckedInAt:     strfmt.DateTime(agent.CreationTimestamp.Time),
+		Role:            role,
+		SuggestedRole:   role,
+		ClusterID:       clusterID,
+		InfraEnvID:      infraEnvID,
+		Status:          &hostStatus,
+		Inventory:       inventory,
+		ValidationsInfo: validationsInfo,
```

Comment:
Worth adding a unit-test for this? I.e. around here: https://github.com/openshift/assisted-service/blob/21d36c5b6b81d480fbfdb39666175abdbf75c964/internal/controller/controllers/agent_controller_test.go#L4291-L4292

Reply:
Definitely! Good idea, thank you! Just added!